README.md

```diff
@@ -95,6 +95,10 @@ bin/deploy -e stage -c --only sul-dlss/dor-services-app
 
 ### Notes and tips:
 * All repos will be cloned to `tmp/repos`.
+* Any repos cloned to `tmp/repos` that are no longer configured in the `settings.yml` file will
+be automatically removed when you deploy or check cocina versions to prevent outdated code from
```

Comment:
or check ssh, but maybe we could get away with looser language like:

```suggestion
* Any repos cloned to `tmp/repos` that are removed from `config/settings.yml`, *e.g.* projects that have been decommissioned, will be automatically removed from `tmp/repos` the next time any of the sdr-depoy commands are run (unless the repo update is explicitly skipped via user-provided flag).
```

or some such